### DISCUSSION TITLE
display HUD on fail

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#17975] [Ideas] [answered]
for the accuracy and health HUD, when you set it to "hide during gameplay", it doesn't show up when you fail. I think it should, because then you can see your accuracy after you fail, which i currently cant do. I don't want to see it while im playing (as its distracting) but you can only check your acc if you have a replay or a submitted score... this would be a very nice QOL feature.

**Comment by peppy** [comment, marked as answer]
I've implemented this via https://github.com/ppy/osu/pull/17977. Do also note that there is a "hold for hud" hotkey which may be useful to you. This defaults to the control key.

**Reply by Axilotl17** [reply]
it doesnt trigger after a fail.

**Reply by peppy** [reply]
It works fine here (during the fail animation). Are you sure?

**Reply by Axilotl17** [reply]
no, i'm not - but either way, because of how i have it set up, its impossible to make a difference without #17978 being implemented (which im exited for)

**Reply by frenzibyte** [reply]
I'm quite curious about that, because all that https://github.com/ppy/osu/pull/17977 achieves is automatically showing the HUD on fail if you had it on "Hide during gameplay" mode.
